endpoints.json

```diff
@@ -73,5 +73,23 @@
         }
       ]
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "serves an updated article",
+    "queries": [],
+    "exampleResponse": {
+      "article": [
+        {
+          "article_id": 1,
+          "title": "Living in the shadow of a great man",
+          "topic": "mitch",
+          "author": "butter_bridge",
+          "body": "I find this existence challenging",
+          "created_at": "2020-07-09T20:11:00.000Z",
+          "votes": 110,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        }
+      ]
```

Comment:
an exampleRequest is needed, to show the user what is the write properties to include in the body